examples/multi-cluster/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+namespace: <namespace here>
+kind: Kustomization
+nameSuffix: -cluster1
+commonLabels:
+  app: cockroachdb-cluster1
+configMapGenerator:
+- envs:
+  - config/cockroach
+  name: cockroach
+- envs:
+  - config/cockroach-backup-config
+  name: cockroach.backup.config
+resources:
+- github.com/utilitywarehouse/cockroachdb-manifests//base?ref=v23.1.10-4
+patches:
+- path: cockroach.yaml
+- path: backup-init-job.yaml
+- path: client.yaml
+- path: init-job.yaml
```

Comment:
You might want to configure your editor to always add newlines at the end of files https://wiki.uw.systems/posts/configure-your-text-editor-to-automatically-add-a-newline-to-the-end-of-a-file-rb9ztniw

Reply:
Yeah. My bad. Thanks for the link 🙇‍♂️